client/src/pages/ForgotPassword/ForgotPassword.tsx

```diff
@@ -0,0 +1,42 @@
+import { useState } from 'react';
+import { Box, Typography, Paper } from '@material-ui/core';
+import ForgotPasswordForm from './ForgotPasswordForm/ForgotPasswordForm';
+import { forgotPassword } from '../../helpers/APICalls/forgotPassword';
+import useStyles from './useStyles';
+
+export default function Reset(): JSX.Element {
+  const [hasSubmitted, setHasSubmitted] = useState<boolean>(false);
+  const [error, setError] = useState<string>();
+  const classes = useStyles();
+
+  const handleSubmit = (
+    { email } : { email: string }
+  ) => {
+    forgotPassword(email)
```

Comment:
If we encounter an error the first time, wouldn't we want to update the error state if the request is successful the next time around?